apps/studio/next.config.mjs

```diff
@@ -37,14 +37,17 @@ const ContentSecurityPolicy = `
     'self'
     https://intercom-sheets.com
     https://www.intercom-reporting.com
-    https://player.vimeo.com
     https://fast.wistia.net
     https://www.google.com
-    https://www.youtube.com
```

Comment:
to confirm: removing `https://www.youtube.com` is not breaking change?

Reply:
Hmm yeah it won't be. I made this change to sync both the CSP here and in our infra repo, plus our component explicitly replaces it with the nocookie version:

https://github.com/opengovsg/isomer/blob/c40ec4b519775f35ac3b690657dddb568e9840f0/packages/components/src/templates/next/components/complex/Video/Video.tsx#L7-L9